src/odemis/driver/technolution.py

```diff
@@ -1131,9 +1131,17 @@ def _acquire(self):
                         logging.warning("ASM acquisition already had the '%s', received this command again." % command)
                         continue
 
-                    acquisition_in_progress = True
                     megafield_metadata = args[0]
-                    self.parent.asmApiPostCall("/scan/start_mega_field", 204, megafield_metadata.to_dict())
+                    notifier_func = args[1]  # Return function (usually, dataflow.notify or acquire_single_field queue)
+                    try:
+                        self.parent.asmApiPostCall("/scan/start_mega_field", 204, megafield_metadata.to_dict())
+                        notifier_func(None)  # return None if all went fine
+                    except Exception as ex:
```

Comment:
The only expected exception here would be the AsmApiException isn't it from asmApiPostCall? Should we not limit the catch to just that exception? Instead of catching all the exceptions? Even the error is already logged in that method.

Reply:
I think that's fine. Normally, only AsmApiExceptions are raised. But if there are other exceptions (ex, the connection failed, division by zero, index out of bound...) the mega field is most likely not started. It's still worth reporting it.